### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by ktsakalozos**
Improve the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,73 @@
...
+```
+
+And bring up the cluster:
+```bash
+sudo k8s bootstrap
+```
+
+For more information and instructions, please see the official documentation at: https://ubuntu.com/kubernetes
+
+### What is included in the k8s distribution
```
Comment (evilnick) [inline README.md:18]:
```suggestion
### What is included in the k8s distribution?
```

Comment (evilnick) [reply README.md:18]: Alternatively, reword it so it sounds less like a question:

### Additional features

File: README.md [hunk 2]
```diff
@@ -1,25 +1,73 @@
 # Canonical Kubernetes Snap
 [![End to End Tests](https://github.com/canonical/k8s-snap/actions/workflows/e2e.yaml/badge.svg)](https://github.com/canonical/k8s-snap/actions/workflows/e2e.yaml)
 
-Canonical Kubernetes is an opinionated Kubernetes delivered by snaps. The focus is on:
-- simplified almost zero operations and
-- enhanced security posture on any infrastructure
+**Canonical Kubernetes** is the fastest, easiest way to deploy a fully-conformant Kubernetes cluster. Harnessing pure upstream Kubernetes, this distribution adds the missing pieces (e.g. ingress, dns, networking) for a zero-ops experience. For single-node clusters it can be deployed with two commands. Add new nodes with just two more. 
 
+Get started in just two commands:
```
Comment (evilnick) [inline README.md:6]: I would skip these instructions as they are basically repeated in better detail below

File: README.md [hunk 3]
```diff
@@ -1,25 +1,73 @@
...
+- a gateway API controller
+- a metrics server
 
-## What are the built-in components?
-The built-in components (network, DNS, storage, RBAC, ingress and gateway) provide most basic functionalities crucial for K8s clusters.
 
 ## Quickstart
 
-Install K8s with:
+Install the Canonical Kubernetes and initialise the cluster with:
```
Comment (evilnick) [inline README.md:33]:
```suggestion
Install Canonical Kubernetes and initialise the cluster with:
```

File: README.md (hunk 2 above)
Comment (bschimke95) [inline README.md:6]: This reads a bit weird.
It either say:
```
Get started with:
sudo snap install k8s --classic
And bring up the cluster:
sudo k8s bootstrap
```
or

Get started in just two commands:
```
sudo snap install k8s --classic
sudo k8s bootstrap
```

File: README.md [hunk 4]
```diff
@@ -1,25 +1,73 @@
...
+
+For more information and instructions, please see the official documentation at: https://ubuntu.com/kubernetes
+
+### What is included in the k8s distribution
+
+In addition to the upstream Kubernetes services, Canonical Kubernetes also includes:
+
+- a DNS service for the node
+- a CNI for the node/cluster
+- a simple local storage provider
```
Comment (bschimke95) [inline README.md:24]: maybe remove `simple`? You haven't used adjectives for the other components.

**Review by bschimke95 — APPROVED** [review]
LGTM, only two comments

File: README.md (hunk 2 above)
Comment (perk) [reply README.md:6]: I like the second suggestion better.

**Pull request merged**